docs/uccignore.md

```diff
@@ -5,4 +5,23 @@ add-on recursively overrides the output folder.
 
 It is expected to be placed in the same folder as `globalConfig` file to have effect.
 
-You will see a warning message in case ignored file is not found in the output folder.
+Uccignore supports wildcard expressions, thanks to which we can find all files matching a specific pattern.
+
+e.g. for given file structure 
+
+```
+...
+└── lib
+    └── 3rdparty
+        ├── linux
+        │   ├── pycache.pyc
+        ├── linux_with_deps
+        │   ├── pycache.pyc
+        └── windows
+            ├── pycache.pyc
+```  
+
+we can remove all `.pyc` files by adding `lib/**/pycache.pyc` to .uccignore file. 
```

Comment:
Let's add a note here saying that it will remove *ALL* pyc files in lib folder even if they are in requests library (for example).